insights_client/__init__.py

```diff
@@ -15,11 +15,17 @@
 import logging.handlers
 from subprocess import PIPE
 
-# setup eggs
+__author__ = 'Richard Brantley <rbrantle@redhat.com>, Jeremy Crafts <jcrafts@redhat.com>, Dan Varga <dvarga@redhat.com>'
```

Comment:
I thought we were removing \_\_author__ and STDOUT_PREFIX

Reply:
Yea those can be :wastebasket:.